packages/lesswrong/components/users/EditProfileForm.tsx

```diff
@@ -61,7 +70,12 @@ const EditProfileForm = ({classes}: {
       <Typography variant="display3" className={classes.heading} gutterBottom>
         Edit Public Profile
       </Typography>
-      <div className={classes.subheading}>All fields are optional</div>
+      <div className={classes.subheading}>
+        All fields are optional.
+        {(terms.slug === currentUser.slug || terms.documentId === currentUser._id) && <span className={classes.importText}>
```

Comment:
Does this need forum gating?

Reply:
This page is only visible on the EA Forum, but I should prob add another gate here just in case another forum enables profiles in general.